pkg/mock/mockcluster/mockcluster.go

```diff
@@ -406,13 +406,23 @@ func (mc *Cluster) UpdateStoreRegionSize(storeID uint64, size int64) {
 	store := mc.GetStore(storeID)
 	newStats := proto.Clone(store.GetStoreStats()).(*pdpb.StoreStats)
 	newStats.Available = newStats.Capacity - uint64(store.GetRegionSize())
+	newStats.UsedSize = uint64(store.GetRegionSize() * mb)
 	newStore := store.Clone(
 		core.SetStoreStats(newStats),
 		core.SetRegionSize(size),
 	)
 	mc.PutStore(newStore)
 }
 
+// UpdateStoreRegionSizeAlone only updates store region size.
+func (mc *Cluster) UpdateStoreRegionSizeAlone(storeID uint64, size int64) {
```

Comment:
prefer to use `UpdateStoreRegionSize` here And change the other one

Reply:
What the name about the other one?

Reply:
How about UpdateStoreSize